src/ontology/uberon-edit.obo

```diff
@@ -224215,6 +224215,1025 @@ relationship: dc-contributor https://orcid.org/0000-0001-6677-8489
 relationship: part_of UBERON:0001114 ! right lobe of liver
 property_value: dcterms-date "2023-05-15T10:52:39Z" xsd:dateTime
 
+[Term]
+id: UBERON:8310000
+name: ureter junction of bladder trigone
+def: "The anatomical junction between the ureter and the bladder trigone. This junction is located at the superior/rostral right and left angles of the base of the bladder trigone on the posterior aspect of the organ where the ureters merge with the bladder wall." [http://scicrunch.org/resolver/SCR_018709]
+xref: EMAPA:36080
+xref: ILX:0738403
+is_a: UBERON:0000061
+
+[Term]
+id: UBERON:8310001
+name: urethra junction with bladder trigone
+def: "The anatomical junction between the urethra and the bladder trigone. This junction is located at the apex of the bladder trigone where the urethra forms." [http://scicrunch.org/resolver/SCR_018709]
+xref: EMAPA:36081
+xref: ILX:0738411
+is_a: UBERON:0000061
+
+[Term]
+id: UBERON:8310002
+name: urethral seam
+def: "A solid epithelial cord present deep to the skin along the ventral midline of the penis. Represents a remnant of a developmental process that closes the urethra." [http://scicrunch.org/resolver/SCR_018709]
+synonym: "urethral raphe" EXACT []
+synonym: "ventral seam" BROAD []
+xref: EMAPA:36421
+xref: ILX:0738407
+is_a: UBERON:0000061
+
+[Term]
+id: UBERON:8310003
+name: anterior subdiaphragmatic vagus nerve
+def: "A nerve formed by the convergence of nerve bundles of the esophageal plexus that arise primarily from the left vagus nerve." [http://scicrunch.org/resolver/SCR_018709]
+xref: ILX:0738436
+is_a: UBERON:0001759
+
+[Term]
+id: UBERON:8310004
+name: interscapular brown adipose tissue
+def: "A collection of brown adipose tissue between the medial borders of the scapulae." [http://scicrunch.org/resolver/SCR_018709]
+synonym: "iBAT" EXACT []
+xref: ILX:0738444
+xref: MA:0003062
+is_a: UBERON:0001348
+
+[Term]
+id: UBERON:8310005
```

Comment:
@MCSZ need a definition source for this one

Reply:
added to Interlex - hasDefinitionSource(Annotation) https://en.wikipedia.org/wiki/Brown_adipose_tissue